src/main/java/net/minecraftforge/common/extensions/IForgeBlock.java

```diff
@@ -772,4 +772,26 @@ else if (state.is(Blocks.OBSERVER))
             return state.isSignalSource() && direction != null;
         }
     }
+
+    /**
+     * Whether this block hides the neighbors face pointing towards this blocks face in the given direction.
+     * <p>
+     * This method should only be used for blocks you don't control, for your own blocks override
+     * {@link Block#skipRendering(BlockState, BlockState, Direction)} on the respective block instead
+     * <p>
+     * WARNING: This method is likely to be called from a worker thread! If you want to retrieve a
```

Comment:
The warning should also include information regarding to CMEs for any kind of write operation.